java/client/src/main/java/glide/managers/RedisExceptionCheckedFunction.java

```diff
@@ -0,0 +1,18 @@
+package glide.managers;
+
+import glide.api.models.exceptions.RedisException;
+
+@FunctionalInterface
+public interface RedisExceptionCheckedFunction<R, T> {
+
+    /**
+     * Functional response handler that takes a protobuf Response object. <br>
```

Comment:
this documentation isn't correct, though - the function doesn't take a protobuf object, it takes `R`.

Reply:
Right. Implementation receives either a response or a long, and this could be implemented to evaluate other value types as needed in the future.

Reply:
updated under 97704828e12fba778b27dd11d2c116e1b173eafd